src/components/Dashboard/DeviceSettingsModal.js

```diff
@@ -354,7 +354,7 @@ class DeviceSettingsModal extends Component {
           { this.props.device.prime &&
             <div className={ classes.unpairWarning }>
               <WarningIcon />
-              <Typography>unpairing will also cancel your prime subscription</Typography>
+              <Typography>Unpairing will also cancel the device's comma prime subscription.</Typography>
```

Comment:
```suggestion
              <Typography>Unpairing will also cancel the comma prime subscription for this device.</Typography>
```

Reply:
I think the current wording is better, but don't prefer it strongly. @pd0wm want to break the tie?

Reply:
Think @gregjhogan wording makes more sense. The device doesn't own the prime subscription.